.github/scripts/label_utils.py

```diff
@@ -111,7 +111,9 @@ def has_required_labels(pr: "GitHubPR") -> bool:
 
 
 def is_label_err_comment(comment: GitHubComment) -> bool:
+    # comment.body_text returns text without markdown
+    no_format_title = LABEL_ERR_MSG_TITLE.replace("`", "")
```

Comment:
Is there a way to test this? I'm assuming we shipped https://github.com/pytorch/pytorch/pull/101747 and then something happened?

Reply:
After the previous PR landed, I noticed a PR with 2 of this comments, while there should be only one.
There is a unit test that failing after this change, I'll update it.